api/src/main/proto/stellarstation/api/v1/stellarstation.proto

```diff
@@ -228,3 +254,54 @@ message Pass {
   // this pass.
   double uplink_center_frequency_hz = 10;
 }
+
+// A plan that specifies antenna configuration for tracking a satellite. A plan
+// is most often created from a pass but does not have to. A plan can contain
+// arbitrary antenna angles and communication frequencies.
+message Plan {
+  // The unique ID of the plan.
+  string id = 1;
+
+  // The id of the satellite to be tracked in the plan.
+  string satellite_id = 2;
+
+  // A description of the status of a plan.
+  enum Status {
```

Comment:
As I understand it, the new API can only ever return 'RESERVED' here for the API we've added. Is that accurate?